tapir/shifts/models.py

```diff
@@ -721,6 +721,35 @@ def get_current_shift_exemption(self, date=None):
     def is_currently_exempted_from_shifts(self, date=None):
         return self.get_current_shift_exemption(date) is not None
 
+    def send_shift_reminder_emails(self):
+        today = datetime.date.today()
+        next_monday = today + datetime.timedelta((0 - today.weekday()) % 7)
+        next_sunday = next_monday + datetime.timedelta(days=7)
+        for attendance in ShiftAttendance.objects.with_valid_state().filter(
+            user=self.user,
+            slot__shift__start_time__gte=next_monday,
+            slot__shift__start_time__lte=next_sunday,
+        ):
+            self.send_shift_reminder_email(attendance.slot.shift)
+
+    def send_shift_reminder_email(self, shift: Shift):
+        template_name = (
+            f"shifts/email/shift_reminder_{self.user.preferred_language}.txt"
```

Comment:
Hm, I realized we are bypassing the Django translation system here. Translations for this email will have to be done outside the usual .po file. Maybe better just put everything in `blocktranslate`?

Reply:
True, I didn't think about it.
I tried putting it in a blocktranslate, the .po files looks like this. It's not super pretty, but maybe still worth it for the sake of following the usual translation way?

`#: shifts/templates/shifts/email/shift_reminder_en.html:1
#, python-format
msgid ""
"\n"
"Hi %(firstname)s,\n"
"\n"
"This is an automated email from SuperCoop.\n"
"\n"
"Your next shift at SuperCoop is this week!\n"
"Your shift is : %(shift_display)s\n"
"\n"
"In case you can't attend the shift, contact the teamleader as soon as "
"possible.\n"
"\n"
"Cooperatively yours,\n"
msgstr ""`

Reply:
As long as it displays properly in Poedit or whatever we use, it's fine I think.